### ISSUE TITLE
D1 messages from outdoor temperature sensor

### ISSUE THREAD

**Issue by bobzomer** [Th3M3/buderus_ems-wiki#3]
I've got a Buderus GB125 heater, with an outdoor temperature sensor. While trying to get information to add it to EMS-ESP project from @proddy, I discovered the outdoor temperature was sent on D1 messages.

Example: `08 00 D1 00 00 80`
The last two bytes (`00 80`) is the outside temperature in 0.1°C (12.8°C in the example).

Could you add it to your wiki so that the documentation is complete?

**Comment by proddy** [comment]
yes, I'm aware of this too. I'll add this along with the new E* messages to my wiki and working with @Th3M3 to consolidate the two

**Comment by proddy** [comment]
https://github.com/proddy/EMS-ESP/wiki/MC110-controller

**Comment by Th3M3** [comment]
@bobzomer:
I already had the `d1` telegram in my wiki [(here)](https://github.com/Th3M3/buderus_ems-wiki/blob/master/Quelle_08.md#typ-0xd1-ubaoutdoortempmessage) and I'm glad to hear that it coincides with your discovery.
